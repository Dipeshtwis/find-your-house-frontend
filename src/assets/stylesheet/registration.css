input {
  outline: none;
  border: none;
}

.input-cl {
  font-size: 15px;
  line-height: 1.5;
  color: #666;
  display: block;
  width: 90%;
  background: #e6e6e6;
  height: 50px;
  border-radius: 25px;
  padding: 0 30px 0 68px;
  margin: 20px;
}

.deauth {
  background: url('../img/white.jpg') no-repeat center;
}

.register-header {
  text-align: center;
  font-size: 34px;
}

.div-btn {
  width: 90%;
  display: flex;
  flex-wrap: wrap;
  justify-content: center;
  padding-top: 20px;
  margin: 20px;
}

.incpass {
  font-size: 18px;
  color: red;
  margin-left: 50px;
}

.form-btn {
  font-size: 15px;
  line-height: 1.5;
  color: #fff;
  text-transform: uppercase;
  width: 100%;
  height: 50px;
  border-radius: 25px;
  background: #ff3d00;
  display: flex;
  justify-content: center;
  align-items: center;
  padding: 0 25px;
  -webkit-transition: all 0.4s;
  -o-transition: all 0.4s;
  -moz-transition: all 0.4s;
  transition: all 0.4s;
}

.text-link {
  text-align: center;
  margin-top: 20px;
  width: 90%;
  font-size: 16px;
}

.text-link a {
  color: black;
}

.text-link a:hover {
  color: #ff3d00;
}

@media screen and (min-width: 1020px) {
  .register-header {
    font-size: 48px;
  }
}
