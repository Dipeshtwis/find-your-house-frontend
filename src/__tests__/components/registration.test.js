import React from 'react';
import renderer from 'react-test-renderer';
import registration from '../../components/Registration';

it('renders registration component correctly', () => {
  const registration = renderer.create(<registration />).toJSON();
  expect(registration).toMatchSnapshot();
});