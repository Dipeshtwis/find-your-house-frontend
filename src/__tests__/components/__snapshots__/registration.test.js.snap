// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders registration component correctly 1`] = `<registration />`;
